src/static/js/app/api/SurveyApi.js

```diff
@@ -1,20 +1,20 @@
 import 'fetch';
 
 export default class SurveyApi {
-  static getFirstPageForUserId(userId, callback) {
-    fetch(`/api/v1/survey/getFirstPage/?userId=${userId}`)
+  static getFirstPageForUserId(userId, quizType, callback) {
+    fetch(`/api/v1/survey/getFirstPage/?userId=${userId}&quizType=${quizType}`)
       .then((response) => response.json())
       .then((jsonData) => callback(jsonData));
   }
 
-  static getSurveyPage(userId, pageNum, callback) {
-    fetch(`/api/v1/survey/${userId}/${pageNum}/`)
+  static getSurveyPage(userId, pageNum, quizType, callback) {
+    fetch(`/api/v1/survey/${userId}/${pageNum}/?quizType=${quizType}`)
       .then((response) => response.json())
       .then((jsonData) => callback(jsonData));
   }
 
-  static submitAnswers(userId, questionData) {
-    fetch(`/api/v1/survey/post/${userId}/`, {
+  static submitAnswers(userId, questionData, quizType) {
```

Comment:
Wtf, `questionData` isn't even in a weird new ES2015 string, it's used in a function call below. C'mon hound.